Source/Core/sampleTerrain.js

```diff
@@ -48,6 +48,78 @@ function sampleTerrain(terrainProvider, level, positions) {
   });
 }
 
+const ACTION__CONTINUE = "continue";
```

Comment:
CesiumJS uses enums for strings like this. However, since there's only two possible states, maybe we could resolve to true or false depending on the state?